api/swagger/swagger.yaml

```diff
@@ -4034,6 +4034,11 @@ paths:
           description: "ID of comment to get"
           required: true
           type: string
+        - name: format
+          in: path
```

Comment:
I think this should be a queryParam not a path in the URL:  foo.com/api/export?format=csv|json